src/pages/classes.scss

```diff
@@ -1,20 +1,36 @@
 .classes {
+  padding: 1rem;
   &__hero {
-    padding: 2rem 1rem;
     max-width: 1170px;
     margin: auto;
     width: 100%;
     color: var(--text-primary);
 
     &__title {
       font-weight: bold;
+      margin-bottom: 0.5rem;
```

Comment:
I say embrace the space!

There is such thing as too many things in a few hundred pixels